src/Microsoft.IdentityModel.Tokens/InternalValidators.cs

```diff
@@ -85,7 +85,7 @@ internal static void ValidateLifetimeAndIssuerAfterSignatureNotValidatedJwt(Secu
 
                 throw LogHelper.LogExceptionMessage(new SecurityTokenUnableToValidateException(
                     validationFailure,
-                    LogHelper.FormatInvariant(TokenLogMessages.IDX10516, kid, exceptionStrings, securityToken, LogHelper.MarkAsNonPII(validLifetime), validIssuer)));
+                    LogHelper.FormatInvariant(TokenLogMessages.IDX10516, kid, LogHelper.MarkAsNonPII(numKeysInTokenValidationParameters), LogHelper.MarkAsNonPII(numKeysInConfiguration), exceptionStrings, securityToken, LogHelper.MarkAsNonPII(validLifetime), validIssuer)));
```

Comment:
consider stacking parameters as above.

Reply:
Fixed! https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1786/commits/27f73791bb1e017be2e7bc1a0b4c25cba31b9a78